GLOSSARY.md

```diff
@@ -0,0 +1,21 @@
+# Glossary
+
+- **Desired state**
+
+    The aggregate of all configuration data for a system form its "Desired State". The "Desired State" of a system is defined as data sufficient to recreate the system from nothing so that instances of the system are behaviorally indinstinguishable.
+
+- **State Store**
+    
+    A system for storing versioned, immutable Desired States that provides access control and auditing on the chnages to the Desired State. Git may be configured as a State Store, but special precautions must be taken.
+
+- **Reconciliation**
+  
+  A continuously running attempt to reconcile _Definition_ of desired state with the current state.
```

Comment:
```The process by which the current state of a system is compared against and made consistent with the system's desired state as declared in the state store```

I think the concept of reconciliation doesn't necessarily mean it's continuous, rather a one off process that looks to achieve consistency between two different states. It can be made continuous, but that is not implicit.